aws-resourceexplorer2-defaultviewassociation/resource-role.yaml

```diff
@@ -34,6 +34,10 @@ Resources:
                 - "resource-explorer-2:DisassociateDefaultView"
                 - "resource-explorer-2:GetDefaultView"
                 Resource: "*"
+              - Effect: Allow
+                Action:
+                - "iam:CreateServiceLinkedRole"
+                Resource: !Sub "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/aws-service-role/resource-explorer-2.amazonaws.com/AWSServiceRoleForResourceExplorer"
```

Comment:
Permissions to create SLR should only be needed in `CreateIndex`.

Reply:
Removed SLR from the DefaultView and View